static/js/core.js

```diff
@@ -9,15 +9,23 @@ function alpineCore() {
         version: '0.0.0',
         isFirstVisit: false,
         scrollTop: window.scrollY,
+        newestVersion: '0.0.0',
+        newVersionAvailable: false,
 
         initPage() {
             window.onscroll = () => {
                 this.scrollTop = window.scrollY;
             };
 
-            apiV2('GET', '/api/v2/health').then((response) => {
+            apiV2('GET', 'https://api.github.com/repos/mitre/caldera/releases').then((response) => {
+                this.newestVersion = response[0].name;
+                return apiV2('GET', '/api/v2/health');
+            }).then((response) => {
                 this.version = response.version;
                 this.checkIfFirstVisit();
+                if (response.version < this.newestVersion) {
```

Comment:
This check needs some work, since the versions are strings not numbers. Here's an example (though it's not written in JS: https://medium.com/@nisargdevdhar/leetcode-solution-227e0b6731ff

I tested it by hard-coding the caldera version to 4.0.0 to see if that newVersionAvailable flag is set to true.